packages/vite/src/node/publicUtils.ts

```diff
@@ -3,7 +3,11 @@
  * This file will be bundled to ESM and CJS and redirected by ../index.cjs
  * Please control the side-effects by checking the ./dist/node-cjs/publicUtils.cjs bundle
  */
-export { VERSION as version } from './constants'
+export {
+  VERSION as version,
+  DEFAULT_DEV_PORT,
+  DEFAULT_PREVIEW_PORT,
```

Comment:
@chenfan0 not sure if it's intentional, but this is still exported.

Reply:
@bluwy I have now removed